template.yaml

```diff
@@ -70,6 +91,8 @@ Resources:
           CodesToOmit: !Ref CodesToOmit
           NoProgramCode: !Ref NoProgramCode
           OtherCode: !Ref OtherCode
+          CacheBucket: !Ref CacheBucket
+          CacheBucketPath: 'mip/coa.json'
```

Comment:
the documents indicate that `CacheBucketPath` is a parameter however it's hard coded here?

Reply:
whoops, good catch, I'm not sure why I listed it as a parameter.